### ISSUE TITLE
A context menu ellipsis … is superfluous where e.g. ▶ signifies a submenu

### ISSUE THREAD

**Issue by grahamperrin** [dessant/search-by-image#42]
The third screenshot at <https://github.com/MrAlex94/Waterfox/issues/708#issuecomment-410153522>, for example.

**Comment by dessant** [comment]
Yeah, this definitely does not look good. I think I'm also going to use the extension name for the context menu item group.

**Comment by grahamperrin** [comment]
Thanks. (A minor issue. I wouldn't have noticed if the Mozilla bug hadn't cropped up.)
